OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitYoutubePlayerFragment.java

```diff
@@ -114,8 +114,20 @@ protected long getPlayerCurrentPosition() {
 
     @Override
     protected void setFullScreen(boolean fullscreen) {
+        /*
+         * If the youtube player is not in a proper state than it throws the  IllegalStateException.
```

Comment:
2 nits:

- then instead of than.
- remove extra space before `IllegalStateException`.

Reply:
updated.